src/xhydro/gis.py

```diff
@@ -344,10 +344,11 @@ def surface_properties(
         bbox=gdf.total_bounds,
     )
 
-    items = list(search.get_items())
+    items = list(search.items())
 
     # Create a mosaic of
-    da = stackstac.stack(items)
+    epsg = int(items[0].properties["proj:code"].split(":")[-1])
```

Comment:
This looks a bit fragile. Is there a way to extract this info using a pyproj class?

Reply:
What exactly? The call itself, or the fact that we use the EPSG from the first item?

Reply:
The call itself. I'm wondering what happens the metadata is older/different. There are definitely helpers for extracting EPSG codes from datasets

Reply:
I can look into it more. If you look at my other changes in that file, it used to be `items.properties["proj:epsg"]` in `pystac <1.12`, but this seems to have been changed to `proj:code`.